File: codechef/septcook0ff17/p1.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long int


int main()
	{
		int t;
		cin>>t;
		while(t--)
		{
			string s;
			cin>>s;
			int a[26];
			memset(a,0,sizeof(a));
			int l=s.length();
			int i,flag=0;
			for(i=0;i<l;i++)
			{
				a[s[i]-'a']++;
				if(a[s[i]-'a']>1){
					flag=1; break;
				}
			}
			if(flag==1)
				cout<<"yes"<<endl;
			else
				cout<<"no"<<endl;
		}
		return 0;
	}

File: codeforces/429div2/p1.py
i=str(raw_input())
k=int(i.split( )[1])
a=str(raw_input())
d=set(a)
flag=0
for i in d:
	if a.count(i) > k: 
		flag=1
if flag==1:
	print 'NO'
else:
	print 'YES'
